tests/main/apt-hooks/task.yaml

```diff
@@ -3,6 +3,18 @@ summary: Ensure apt hooks work
 # apt hook only available on 18.04+ and aws-cli only for amd64
 systems: [ubuntu-18.04-64, ubuntu-2*-64]
 
+prepare: |
+    cp /etc/systemd/system/snapd.service.d/local.conf /etc/systemd/system/snapd.service.d/local.conf.bak
+    # disable testing so that catalog refresh requests are sent out
+    sed 's/SNAPPY_TESTING=1/SNAPPY_TESTING=0/g' -i /etc/systemd/system/snapd.service.d/local.conf
```

Comment:
we cannot do this as a pattern, one of the effect of having this set is that the store doesn't count the operations in its statistics, we need to either use a different env var that effects only catalog refresh or use different values for SNAPP_TESTING instead of it being just a bool

Reply:
Another simpler idea is to support the combination  SNAPPY_TESTING=1  and SNAPD_CATALOG_REFRESH=1

Reply:
Tweaked this now.